cks/templates/cks-keys-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.virtruKeys.name }}
+type: Opaque
+data:
+  rsa001.pub: {{ .Values.appSecrets.virtruKeys.data.rsa001pub | quote }}
+  rsa001.pem: {{ .Values.appSecrets.virtruKeys.data.rsa001pem | quote }}
```

Comment:
why is this template still here if you're already creating the secrets above via kubectl? virtruKeys.data isn't in values.yaml so won't this fail?

Reply:
Oh duh. Deleted and pushed a new commit